docs/source/guide/release_notes.md

```diff
@@ -8,74 +8,72 @@ meta_title: Release notes for Label Studio Enterprise
-This release of Label Studio Enterprise includes fixes for the following bugs and other improvements.
-
-- Instructions were not appearing for annotators before labeling in some cases.
-- Annotators logging in for the first time with LDAP accounts saw a runtime error. <!--HTX-2169-->
-- The "Review Finished" modal appeared before annotation reviews were complete. <!--HTX-2152--> 
-- Using the Naive matching metric for brush annotation projects led to unexpected behavior when saving annotations. <!--HTX-2144-->
-- Creating users using the API did not work as expected. <!--HTX-2133--> 
-- Specifying the `completed_by` user when creating an annotation with the API did not work as expected. <!--HTX-2130-->
-- There was an issue with the `Paragraphs` tag. <!--HTX-2119--> 
-- There was an issue with labeling only selected tasks. <!--HTX-2110-->
-- Brush strokes did not change size when zooming in on an image for labeling.  
-- Improved the cards that display to reviewers and annotators. <!--HTX-1997-->
-- Submitting predictions as annotations did not work as expected. <!--HTX-1990--> 
-- Hotkeys stopping working after submitting an annotation. <!--HTX-1976-->
-- The `selected` parameter did not work properly for the `<Label>` tag. <!--HTX-1945--> 
-- There was an issue submitting and loading annotations with relations. <!--HTX-1708--> 
-- Updating an annotation in the review stream did not work as expected. <!--HTX-1677--> 
\ No newline at end of file
+!!! info 
+    <i class='ent'></i> 
+    The release notes for Label Studio Community Edition is available on the <a href="https://github.com/heartexlabs/label-studio/releases"> Label Studio GitHub repository</a>.
+
+
+## Label Studio Enterprise 2.2.9  
+This section highlights the new features and enhancements, breaking changes, and bug fixes in Label Studio Enterprise 2.2.9. 
+
+### New features and enhancements 
+Label Studio Enterprise 2.2.9 introduces the following new features and enhancements.
+
+#### GUI
+- This release includes the ability to select model version on model cards from the machine learning page >> **Settings**. [DEV-1868](https://heartex.atlassian.net/browse/DEV-1868).
+ 
+#### Templates
+- This release introduces [Dynamic Labels templates](https://labelstud.io/templates/gallery_dynamic_labels.html#main). You can now show labels dynamically as a task input, either as a prediction from the model, or from a database lookup. With a broader set of options, dynamic labeling saves time and increases the consistency of labeling the objects. [DEV-2636](https://heartex.atlassian.net/browse/DEV-2636). 
+
+### Breaking changes
+Label Studio Enterprise 2.2.9 includes the following breaking change.
+
+- This release decommissions MinIO as a service. [DEV-2600](https://heartex.atlassian.net/browse/DEV-2600).
+
+### Bug fixes
+Label Studio 2.2.9 includes the following bug fixes.
+
+- Fixed empty agreement for taxonomy with extra labels. [DEV-2440](https://heartex.atlassian.net/browse/DEV-2440).
```

Comment:
I don't see any reason to keep DEV-NNN URLs here if they are not accessible.

Reply:
https://heartex.atlassian.net/browse/DEV-2440 is not accessible?

All the URLs are working and they are added to the release notes based on our Slack discussion.
<img width="1463" alt="Screen Shot 2022-06-29 at 2 58 24 PM" src="https://user-images.githubusercontent.com/107003313/176552520-026f78eb-4fac-41db-8d89-2fe7822e55fb.png">

Reply:
That was only about ticket numbers (DEV-XXX), from where the URLS links could be guessed, URL links themselves are useless

Reply:
Does this mean, all the URL links should be removed and only the ticket numbers should be listed in the release notes?

Reply:
https://user-images.githubusercontent.com/107003313/176783871-7b49f3c8-6de9-4649-a541-7c26e316a6bc.mov

@niklub @makseq ^^